app/src/main/java/com/dimelo/sampleapp/RcConfigurationActivity.java

```diff
@@ -31,9 +38,38 @@ public void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(myToolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
         textInputLayout = findViewById(R.id.user_id);
+        userNameInputLayout = findViewById(R.id.user_name);
+        companyInputLayout = findViewById(R.id.company);
+        emailInputLayout = findViewById(R.id.email);
+        firstnameInputLayout = findViewById(R.id.firstname);
+        lastnameInputLayout = findViewById(R.id.lastname);
+        homePhoneInputLayout = findViewById(R.id.homephone);
+        mobilePhoneInputLayout = findViewById(R.id.mobilephone);
```

Comment:
Please add some clean localized strings for these inputs

Reply:
7267aad6bf0ca71a050cbd6ae37249737f3357e5